packages/web/src/components/gcds-footer/gcds-footer.tsx

```diff
@@ -66,29 +61,51 @@ export class GcdsFooter {
     return !!this.el.querySelector('[slot="list"]');
   }
 
+  private get hasContextual() {
+    return !!this.el.querySelector('[slot="contextual"]');
+  }
+
   render() {
-    const { lang, display, topHref, hasList, renderSignature } = this;
+    const { lang, display, hasList, hasContextual, renderSignature } = this;
     const govNav = I18N[lang].gov.menu;
+    const themeNav = I18N[lang].themes.menu;
     const siteNav = I18N[lang].site.menu;
 
     return (
       <Host role="contentinfo">
         <div class="top__container">
           <slot name="top"></slot>
         </div>
-
+        {hasContextual && 
+          <div class="gcds-footer__contextual">
+            <div class="contextual__container">
+              <slot name="contextual"></slot>
+            </div>
+          </div>}
         {display === "full" ?
           (<div class="gcds-footer__landscape">
-            <nav class="landscape__container" aria-label={I18N[lang].gov.heading}>
-              <h2>{I18N[lang].gov.heading}</h2>
-              <ul>
-                {Object.keys(govNav).map((value) =>
-                  <li>
-                    <a href={govNav[value].link}>{govNav[value].text}</a>
-                  </li>
-                )}
-              </ul>
-            </nav>
+            <div class="landscape__container">
+              <nav class="landscape__govnav" aria-label={I18N[lang].gov.heading}>
+                <h3>{I18N[lang].gov.heading}</h3>
+                <ul>
+                  {Object.keys(govNav).map((value) =>
+                    <li>
+                      <a href={govNav[value].link}>{govNav[value].text}</a>
+                    </li>
+                  )}
+                </ul>
+              </nav>
+              <nav class="landscape__themenav" aria-label={I18N[lang].themes.heading}>
```

Comment:
In the example footer, both of these lists are in the same nav element. Did you split them up into 2 nav's to have them both show up as landmarks?

Reply:
Yes I decided to split the larger nav element into two different nav elements, since the first links are more global in context and the themes and topics are more specific.

Do you think I should format it the way canada.ca does or do you think this way will also work?